tests/sanity/ignore-2.10.txt

```diff
@@ -1,3 +1,4 @@
+plugins/action/iptables_state.py action-plugin-docs
```

Comment:
This one should be unnecessary after the rename + adding the symlink.

Reply:
Yes it is (and triggers an error).
All suggestions applied :) thanks